### ISSUE TITLE
adding reset password

### ISSUE THREAD

**Issue by Zircoz** [Zircoz/express-locallibrary-tutorial#4]
Prerequisite for this feature: #5 
- Reset password for forgotten passwords
- Reset password when logged in using current password

**Comment by singhsaurav23** [comment]
I would like to work on this issue. Please assign this to me.
I would be grateful if you add a DWOC tag here.

**Comment by Zircoz** [comment]
@singhsaurav23 yes i will add a contribution guidelines page and edit the readme, then this issue will be assigned to you ! Stay tuned !

**Comment by singhsaurav23** [comment]
@Zircoz  ok sure!!. Please don't forget to add the DWOC label tag
